packages/dom/src/dom/is-edge.js

```diff
@@ -16,11 +16,11 @@ import isInputOrTextArea from './is-input-or-text-area';
  *
  * @param {Element} container    Focusable element.
  * @param {boolean} isReverse    Set to true to check left, false to check right.
- * @param {boolean} onlyVertical Set to true to check only vertical position.
+ * @param {boolean} [onlyVertical=false] Set to true to check only vertical position.
```

Comment:
Nit: Can we keep the vertical alignment of these descriptions?

Reply:
Soooo, strong opinion that I'm happy to re-evaluate about vertical alignment: (I think this is probably controversial in the WP community) but I use a proportional font in my editor as it's about 10 times easier for me to read code that way, so I have no idea when code is vertically aligned or not. Given we also use tab stops instead of spaces to accomodate a variety of tab widths, it's odd that we ever vertically align code to me because it also seems to preference a specific style of editor.

So what's say we drop the vertical alignment altogether and I just make the uniformly not-aligned?

Like I said, happy to re-evaluate this opinion, I don't personally see a benefit to vertical alignment but maybe there's something big I'm missing.

Reply:
Vertical alignment is an essential code reading tool I'm using in my editor, but I'll agree to disagree in this instance and leave this for another discussion 😉  Feel free to merge as it is.